server.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const bcrypt = require('bcryptjs')
+const session = require('express-session')
+const server = express();
+const DB = require('./data/dbHELPER.js')
+
+server.use(express.json())
+server.use(session({
+    name: 'notsession', // default is connect.sid
+    secret: 'nobody tosses a dwarf!',
+    cookie: {
+      maxAge: 1 * 24 * 60 * 60 * 1000,
+    }, 
+    httpOnly: true, // don't let JS code access cookies. Browser extensions run JS code on your browser!
+    resave: false,
+    saveUninitialized: false,
+  }))
+
+const PORT = 4444;
+
+server.get('/api/users', (req, res) => {
+    if (req.session && req.session.userID){
```

Comment:
Instead of the if statement on line 22, you can create a middleware function that is only allows users with sessions signed in to access a get endpoint. That way if you have multiple endpoints that are restricted to signed in users, you can re use the middleware